cmd/kyverno/main.go

```diff
@@ -470,6 +473,14 @@ func main() {
 	kubeInformer := kubeinformers.NewSharedInformerFactory(kubeClient, resyncPeriod)
 	kubeKyvernoInformer := kubeinformers.NewSharedInformerFactoryWithOptions(kubeClient, resyncPeriod, kubeinformers.WithNamespace(config.KyvernoNamespace()))
 	kyvernoInformer := kyvernoinformer.NewSharedInformerFactory(kyvernoClient, resyncPeriod)
+	// this label need to be hard-coded as at this point we don't have any resource to get this value from
+	labelsMap := map[string]string{"cache.kyverno.io/enabled": "true"}
```

Comment:
Checking the label is present should be enough (no need to check the value).

Reply:
I also wonder if we want this logic to live somewhere in the `resolvers` package.

Reply:
I couldn't figure out how to use label only. `labelSelector` needs data of type `map[string]string` hence added value as well

Reply:
There is `labels` package to do that. See this https://github.com/kyverno/kyverno/blob/a09d2a77e5619a7b2aba4cd86e3e8fa9ace0da6b/pkg/utils/report/selector.go#L28-L35 for an example.

Reply:
fixed